setup.py

```diff
@@ -58,8 +58,10 @@ def run(self):
 
         if self.type in ['unit', 'all']:
             cmd.append('python/Ganga/new_tests/Unit')
-            cmd.append('python/Ganga/Core')
-            cmd.append('python/Ganga/Runtime')
+            ## Ganga/Core finds zero tests
+            #cmd.append('python/Ganga/Core')
```

Comment:
Some tests like doctests (for example `Ganga/Runtime/bootstrap.py:new_version_format_to_old`) will need to be located alongside their source code. In general we will be locating all tests in the `[new_]tests` directory but for some components it might make sense to put them alongside the code they're associated with.